ClientSide/Benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2018 Vladimir Balun
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required (VERSION 3.10)
+
+project (RacingWorldBenchmarks)
+
+set (BENCHMARKS SIMDVectors)
+
+foreach (BENCHMARK ${BENCHMARKS})
+    add_executable (${BENCHMARK} ./${BENCHMARK}.cpp)
+    target_include_directories (${BENCHMARK} PRIVATE ${PROJECT_SOURCES_DIR}/Math)
+    target_include_directories (${BENCHMARK} PRIVATE ${PROJECT_BENCHMARKS_DIR}/Helpers)
+    #target_include_directories (${BENCHMARK} PRIVATE ${PROJECT_DEPENDENCIES_DIR}/Catch/Include)
```

Comment:
You need to combine lines 25 and 26 into one and remove the unnecessary comment on line 67